.make/Makefile.deploy.prow

```diff
@@ -54,16 +56,16 @@ deploy-plugins: build-plugin-images push-plugin-images
 .PHONY: build-plugin-images $(PLUGINS)
 build-plugin-images: $(BUILD_IMAGES)
 $(BUILD_IMAGES): build-%: %
-	$(DOCKER) build --build-arg BINARY=$< -t $(REGISTRY)/$(DOCKER_REPO)/$< -f $(DEPLOY_DOCKERFILE) .
-	$(DOCKER) tag $(REGISTRY)/$(DOCKER_REPO)/$< $(REGISTRY)/$(DOCKER_REPO)/$<:$(TAG)
+	$(DOCKER) build --build-arg BINARY=$< -t $(REGISTRY)/$(IMG_REPO)/$(OC_PROJECT_NAME)-$< -f $(DEPLOY_DOCKERFILE) .
+	$(DOCKER) tag $(REGISTRY)/$(IMG_REPO)/$(OC_PROJECT_NAME)-$< $(REGISTRY)/$(IMG_REPO)/$(OC_PROJECT_NAME)-$<:$(TAG)
```

Comment:
Am I missing something or does the combination of:
https://github.com/arquillian/ike-prow-plugins/pull/218/files#diff-66536a55e28a60c7e24d2d4a43a8b89cR64
`export IMG_REPO="openshiftio/rhel-"`
with 
`$(REGISTRY)/$(IMG_REPO)/$(OC_PROJECT_NAME)-$<`
result in
`quay.io/openshiftio/rhel-/ike-prow-plugins-test-keeper`
which is not correct

Reply:
@hemanik Is there a fix coming for it though? I can't see updates and would love to close this off.

Reply:
@bartoszmajsak, will update it.